src/pages/resources/fundamentals/_tbd_manifest/index.md

```diff
@@ -0,0 +1,2 @@
+#
+Mor eof a plugin concept but scripts have an invisible manifest. with predefined values
```

Comment:
@anoopvalomkot and @kareemmyk  What are the values set for the permission module for UXP scripts?

Reply:
@padmkris123 
InDesign UXP Scripts are run in trusted (UXP term) mode, which should translate to full permission. This was done to keep it on par with ExtendScript.

Reply:
@padmkris123 
From UXP's side these are the permissions. But do we really want to expose it to the 3Ps? It's fine for first party developers

```
        permissions.network = scriptPermissions.network;
        permissions.clipboard = scriptPermissions.clipboard;
        permissions.launchProcess = scriptPermissions.launchProcess;
        permissions.ipc.enableHostCommunication = scriptPermissions.ipc.enableHostCommunication;
        permissions.ipc.enablePluginCommunication = scriptPermissions.ipc.enablePluginCommunication;
        permissions.localFileSystem = scriptPermissions.localFileSystem;

Reply:
@anoopvalomkot  and @kareemmyk I need exact values for each permisson module. 
For example, what do we mean by 'full' permission for module like 'launchProcess' which requires schemes and extensions?

Reply:
Certain APIs work with a combination of manifest values. If its about to be used within scripts, I should know whether or not it will work based on the manifest.

Reply:
@padmkris123, please refer [3P Plugin Permission Models Alignment](https://wiki.corp.adobe.com/display/UXP/3P+Plugin+Permission+Models+Alignment) for the full permission details

Reply:
@padmkris123 I was answering for UXP scripting and not plugins. @kareemmyk 's response looks appropriate for plugins.

Reply:
@kareemmyk  Anoop is right. I am looking for values for scripts. These values are set and cannot be altered. The wiki only has plugin permission.

Reply:
I have started slack discussion in below thread to collect this information from UXP team.
https://adobeindesign.slack.com/archives/C032A8TM50A/p1690866191113649